docs/gno-tooling/cli/faucet.md

```diff
@@ -0,0 +1,46 @@
+---
+id: gno-tooling-tm2-faucet
```

Comment:
I'm not sure what the use of this file is, it can become outdated very quickly (help output)

Reply:
Removed it: [fbb91ed](https://github.com/gnolang/gno/pull/2122/commits/fbb91edb529b4e9c3032d988bca5b86b6a310d97)